specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2016-05-16/analysisservices.json

```diff
@@ -553,13 +553,6 @@
             "modelAsString": true
           },
           "description": "The name of the Azure pricing tier to which the SKU applies."
-        },
-        "capacity": {
```

Comment:
Was the content removed from this swagger because it didn't accurately reflect the service?